src/NLPModels.jl

```diff
@@ -160,6 +160,22 @@ function jac_op(nlp :: AbstractNLPModel, x :: Vector{Float64})
                         v -> jtprod(nlp, x, v))
 end
 
+"""`J = jac_op!(nlp, x, Jv, Jtv)`
+
+Return the Jacobian at `x` as a linear operator.
+The resulting object may be used as if it were a matrix, e.g., `J * v` or
+`J' * v`. The values `Jv` and `Jtv` are used to preallocate the memory for the
```

Comment:
Could you just change this to "... are used as preallocated storage for the ..."?